pkg/api/jobservice/jobservice.proto

```diff
@@ -0,0 +1,22 @@
+syntax = 'proto3';
+
+package jobservice;
+
+message JobServiceRequest {
+    string job_id = 1;
+    string job_set_id = 2;
+    string queue = 3;
+}
+
+message JobServiceResponse {
+// This is a string representation of the event name
+// See event.proto for a list of possible events
+    string state = 1;
```

Comment:
I guess it's a toss up here between using a string and a proto enum.  I'm mainly in favour of using an enum as it tightens up the api, but let's see what others think.

Reply:
Yea I think that is a good idea. I'll add enums for the events plus unknown.